pygac/data/calibration.json

```diff
@@ -58,7 +58,7 @@
             "dark_count": 41.8,
             "gain_switch": 502.0,
             "s0": 0.1217142857142857,
```

Comment:
Just stumbled upon this: Seems like `s0` for channel 1/2/3a is also slightly different from the PATMOS-x file?

Reply:
Yes, this is because we do not differentiate between high and low gain coefficients, which actually do not exist. I don't know why PATMOS-x does it, but the fitting procedure as described in Heidinger et al 2010 produces the coefficients that we store in our coefficient files. There is a very simple conversion rule to get the low and high gain coefficients as noted in our code
https://github.com/pytroll/pygac/blob/3162eddd6ad3b9285304990d99b35537bb48f9d1/pygac/calibration.py#L278-L295

In order to get the original `s0` parameter, we do the inverse transformation, which requires a bisection to become the exact inverse to the given precision, see `Translator.find_s0`.

The lines
https://github.com/pytroll/pygac/blob/3162eddd6ad3b9285304990d99b35537bb48f9d1/pygac/calibration.py#L244-L251
in the solar calibration will exactly recover the values that you see in the PATMOS-x files (note the rounding in the bisection and in the above lines).

Reply:
Furthermore, note that this only holds for AVHRR/3 instruments (NOAA-15 onward) older instruments do not use single gain counts and therefore `s0` will be the same for low and hig gain and also the same value in our json file.

Reply:
Wow, so much detailed work in there! Thanks for the explanation!